devel/ninja/ninja-cmake.mk

```diff
@@ -0,0 +1,13 @@
+include $(INCLUDE_DIR)/cmake.mk
```

Comment:
If you include `ninja.mk` here too, projects can simply switch from `cmake.mk` to `cmake-ninja.mk` without including both

Reply:
I can't seem to get this working. I'm getting a too many open files error.

Reply:
Never mind.